pkg/views/workspace/create/configuration.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package create
+
+import (
+	"errors"
+	"log"
+
+	"github.com/charmbracelet/huh"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/daytonaio/daytona/pkg/serverapiclient"
+	"github.com/daytonaio/daytona/pkg/views"
+	selection "github.com/daytonaio/daytona/pkg/views/workspace/selection"
+)
+
+var configurationHelpLine = lipgloss.NewStyle().Foreground(views.Gray).Render("enter: next  f10: configuration view")
```

Comment:
I'd label `f10` as `advanced configuration`. Might be more clear that it's intended for "advanced" usage.